docs/modules/Array.ts.md

```diff
@@ -1384,6 +1398,18 @@ assert.deepStrictEqual(pipe([2, 3, 4], prepend(1)), [1, 2, 3, 4])
 
 Added in v2.10.0
 
+## prependW
+
+Less strict version of [`prepend`](#prepend).
+
+**Signature**
+
+```ts
+export declare const prependW: <A, B>(head: B) => (tail: A[]) => NEA.NonEmptyArray<A | B>
```

Comment:
Technically return type can be `[B, ...A[]]`, but suspect that's not particularly useful.